src/standard/utils.html

```diff
@@ -306,6 +306,17 @@
         }
       }
       return elt;
+    },
+
+    /**
+     * Checks whether an element is in this element's shadow DOM tree.
+     *
+     * @method isShadowDomChild
+     * @param {HTMLElement=} node The element to be checked.
+     * @return {Boolean} true if node is in this element's shadow DOM tree.
+     */
+    isShadowDomChild: function(node) {
+      return Polymer.dom(this).getOwnerRoot() !== Polymer.dom(node).getOwnerRoot();
```

Comment:
I thiiiink this will return true for somebody else's local child. So if I have a `paper-item`
```
<paper-item>
  <span></span>
  <paper-checkbox></paper-checkbox>
</paper-item>
```
I believe with the current method, `paperItem.isShadowDomChild(paperCheckbox.someShadowChild)` will return true, and shouldn't.

Hence `this.contains(node)` or something like that.